samples/angular/src/app/i18n/jss-translation-loader.service.ts

```diff
@@ -1,29 +1,18 @@
 import { Injectable } from '@angular/core';
 import { TranslateLoader } from '@ngx-translate/core';
-import { Observable } from 'rxjs';
-import { HttpClient } from '@angular/common/http';
```

Comment:
Remove `HttpClient` mention / rework doc section for `JssTranslationLoaderService` here: https://jss.sitecore.com/docs/client-frameworks/angular/sample-app#jsstranslationloaderservice.

Reply:
Done